package.json

```diff
@@ -17,16 +17,19 @@
 	},
 	"dependencies": {
 		"@darkflorist/address-metadata": "0.2.0",
-		"@noble/curves": "0.8.0",
+		"@noble/curves": "1.1.0",
 		"@noble/hashes": "1.2.0",
+		"@noble/secp256k1": "2.0.0",
 		"@preact/signals": "1.1.3",
+		"@preact/signals-core": "1.3.0",
 		"ethers": "6.4.0",
 		"funtypes": "5.0.3",
 		"preact": "10.8.1",
 		"webextension-polyfill": "0.10.0"
 	},
 	"devDependencies": {
 		"@types/firefox-webext-browser": "109.0.0",
+		"@types/node": "^18",
```

Comment:
Adding `@types/node` needs extra care to be included in a web project, else you won't get compiler errors on trying to reference Node-only globals and imports from the web project (which will be runtime failures).  I think thus far we have managed to avoid it.